extensions/kubeconfig/podlogs.go

```diff
@@ -12,8 +14,14 @@ import (
 	restclient "k8s.io/client-go/rest"
 )
 
-// GetPodLogs is a helper function that resturns logs from a pod using rest client
-func GetPodLogs(client *rancher.Client, clusterID string, podName string, namespace string) (string, error) {
+// GetPodLogs is a function that fetches the logs of a specific pod within a
+// Kubernetes cluster. It requires a rancher client instance, clusterID,
```

Comment:
I'm not completely sure, but do we want to include the requirements in the comment?

Reply:
Updated the comment to be a lot shorter - thanks for the suggestion, it's cleaner 😁✨